gulpfile.js

```diff
@@ -0,0 +1,37 @@
+const gulp = require('gulp');
+const gutil = require('gulp-util');
+const typescript = require('gulp-typescript');
+const sourcemaps = require('gulp-sourcemaps');
+const tsProject = typescript.createProject('tsconfig.json');
+
+const OPTIONS = {
+	header: true,
```

Comment:
I use an editor which decides what to use based on what it sees first. And I think I transformed this originally from a CS file, so that's why. This is why we have reviews. ;-)